presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java

```diff
@@ -52,21 +63,42 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
         this.queuedTime = requireNonNull(queuedTime, "queuedTime is null");
     }
 
+    /**
+     * Returns the coordinator's uri.
+     *
+     * @return an Option {@link CoordinatorLocation}.
+     */
     public Optional<CoordinatorLocation> getCoordinatorLocation()
     {
         return coordinatorLocation;
     }
 
+    /**
+     * If there was a failure, it returns detailed information such as the type, the message
+     * and cause.
+     *
+     * @return an Optional {@link ExecutionFailureInfo}.
+     */
     public Optional<ExecutionFailureInfo> getFailureInfo()
     {
         return failureInfo;
     }
 
+    /**
+     * Returns the elapsed time of the query.
+     *
+     * @return a {@link Duration} object.
+     */
     public Duration getElapsedTime()
     {
         return elapsedTime;
     }
 
+    /**
+     * Returns the queued time.
```

Comment:
I always need to know what this time is related to:
"Returns the queued time of the query."